### PR TITLE
creating new release: 0.0.1-2337

### PR DESCRIPTION
**Pull request by biplamal**

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -1,6 +1,37 @@
 # Ixia-c Release Notes and Version Compatibility
 
-## Release v0.0.1-2289 (Latest)
+
+## Release v0.0.1-2337 (Latest)
+> 21st October, 2021
+
+#### About
+
+This build contains performance improvement in SetConfig and more correct fps vaules in scenarios involving multiple start or stop traffic.
```
Comment (ashutshkumr) [inline docs/releases.md:9]: This build contains bugfixes for `SetConfig` and FPS values in `GetMetrics`.

File: docs/releases.md [hunk 2]
```diff
@@ -1,6 +1,37 @@
...
+| ixia-c-traffic-engine         | [1.4.0.13](https://hub.docker.com/r/ixiacom/ixia-c-traffic-engine/tags)       |
+| ixia-c-app-usage-reporter     | [0.0.1-37](https://hub.docker.com/r/ixiacom/ixia-c-app-usage-reporter/tags)      |
+| ixia-c-protocol-engine        | 1.00.0.83      | 
+| otg-gnmi-server               | [0.6.6](https://hub.docker.com/r/otgservices/otg-gnmi-server/tags)         |
+| otg-grpc-server               | [0.6.6](https://hub.docker.com/r/otgservices/otg-grpc-server/tags)         |
+
+
+#### New Feature(s)
+
+* SetConfig Performance is improved through concurrent operations.
```
Comment (ashutshkumr) [inline docs/releases.md:27]: The race condition during connection initialization in `SetConfig` is fixed for scenarios involving large port count.

File: docs/releases.md [hunk 3]
```diff
@@ -1,6 +1,37 @@
...
+* FPS shows more correct values in scenarios involving multiple start or stop traffic in same test.
```
Comment (ashutshkumr) [inline docs/releases.md:28]: FPS value in `GetMetrics` for ports and flows is fixed for scenarios involving multiple consecutive `SetTransmitState` calls.

File: docs/releases.md [hunk 4]
```diff
@@ -1,6 +1,37 @@
...
+#### Build Details
+
+| Component                     | Version       |
+|-------------------------------|---------------|
+| Open Traffic Generator API    | [0.6.5](https://redocly.github.io/redoc/?url=https://github.com/open-traffic-generator/models/releases/download/v0.6.5/openapi.yaml)         |
+| snappi                        | [0.6.5](https://pypi.org/project/snappi/0.6.5)        |
+| ixia-c-controller             | [0.0.1-2337](https://hub.docker.com/r/ixiacom/ixia-c-controller/tags)    |
+| ixia-c-traffic-engine         | [1.4.0.13](https://hub.docker.com/r/ixiacom/ixia-c-traffic-engine/tags)       |
+| ixia-c-app-usage-reporter     | [0.0.1-37](https://hub.docker.com/r/ixiacom/ixia-c-app-usage-reporter/tags)      |
+| ixia-c-protocol-engine        | 1.00.0.83      |
```
Comment (ashutshkumr) [inline docs/releases.md:20]: also add operator version since it was requested by customers.

File: docs/news.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
 # News
 
+* **21st October, 2021**: Ixia-c version 0.0.1 (build 2337) released.  This conforms to **[v0.6.5](https://redocly.github.io/redoc/?url=https://github.com/open-traffic-generator/models/releases/download/v0.6.5/openapi.yaml)** of the Open Traffic Generator Models specification.
+
+  * This build contains performance improvement in SetConfig and more correct fps vaules in scenarios involving multiple start or stop traffic. [Read more](releases.md)
```
Comment (ashutshkumr) [inline docs/news.md:5]: change this based on my comments on release notes.

File: docs/releases.md (hunk 4 above)
Comment (biplamal) [reply docs/releases.md:21]: Done

File: docs/releases.md [hunk 6]
```diff
@@ -1,6 +1,37 @@
...
+#### About
+
+This build contains performance improvement in SetConfig and more correct fps vaules in scenarios involving multiple start or stop traffic.
+
+#### Build Details
+
+| Component                     | Version       |
+|-------------------------------|---------------|
+| Open Traffic Generator API    | [0.6.5](https://redocly.github.io/redoc/?url=https://github.com/open-traffic-generator/models/releases/download/v0.6.5/openapi.yaml)         |
+| snappi                        | [0.6.5](https://pypi.org/project/snappi/0.6.5)        |
```
[the remark this answers is not in the document]
Comment (biplamal) [reply docs/releases.md:16]: Done

**Pull request merged**